lib/phoenix/endpoint.ex

```diff
@@ -696,8 +696,16 @@ defmodule Phoenix.Endpoint do
       def static_path(path) do
         Phoenix.Config.cache(__MODULE__, :__phoenix_static__,
                              &Phoenix.Endpoint.Supervisor.static_path/1) <>
+        elem(static_lookup(path), 0)
+      end
+
+      def static_integrity(path) do
```

Comment:
We should document this one and the next one (if we decide to keep it public).

Reply:
Ok will add docs.